docs/src/main/sphinx/develop/client-protocol.md

```diff
@@ -9,50 +9,47 @@ It can also be used to implement your own client, if necessary.
 ## HTTP methods
 
 - A `POST` to `/v1/statement` runs the query string in the `POST` body,
-  and returns a JSON document containing the query results.  If there are more
+  and returns a JSON document containing the query results. If there are more
   results, the JSON document contains a `nextUri` URL attribute.
 - A `GET` to the `nextUri` attribute returns the next batch of query results.
 - A `DELETE` to `nextUri` terminates a running query.
 
 ## Overview of query processing
 
 A Trino client request is initiated by an HTTP `POST` to the endpoint
-`/v1/statement`, with a `POST` body consisting of the SQL query string.
-The caller may set various {ref}`client-request-headers`. The headers are
-only required on the initial `POST` request, and not when following the
-`nextUri` links.
+`/v1/statement`, with a `POST` body consisting of the SQL query string. The
+caller may set various [](client-request-headers). The headers are only
```

Comment:
Square brackets are missing the link text

Reply:
Sphinx inserts the title automatically now